File: ELearningBackEnd/src/main/java/com/example/elearningbackend/chat/message/ChatMessageReq.java
package com.example.elearningbackend.chat.message;

import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class ChatMessageReq {

    private long chatRoomId;
    private long senderId;
    private long receiverId;
    private String message;
    private String type;
}

File: ELearningFrontEnd/src/app/dtos/chat/chat-message.req.ts
export class ChatMessageReq{

    chatRoomId: number;
    senderId: number;
    receiverId: number;
    message: string;
    type: string;

    constructor(chatRoomId: number, senderId: number, receiverId: number, message: string, type: string){
        this.chatRoomId = chatRoomId;
        this.senderId = senderId;
        this.receiverId = receiverId;
        this.message = message;
        this.type = type;
    }
}